src/test/scala/com/microsoft/ml/spark/cognitive/split1/TextAnalyticsSDKSuite.scala

```diff
@@ -103,16 +138,27 @@ class TextSentimentSuiteV4 extends TestBase with DataFrameEquality with TextKey
       data(0).get(2).toString == "negative")
     assert(data(1).get(0).toString == "positive")
   }
-}
 
+  test("Sentiment Analysis - Assert Confidence Score") {
+    val replies = detector.transform(batchedDF)
+      .select("output.result.sentiment", "output.result.confidenceScores")
+      .collect()
+    val confidence = replies.map(row => row.getList(1))
+    assert(confidence(0).get(0).toString > "[0.50, 0.0, 0.0]")

```

Comment:
We would like to only compare the positive/negative/neutral positive score which is larger than 0.5 right? Not the whole list.  The ToString comparison won't make sense?